packages/spacecat-shared-data-access/test/unit/models/api-key.test.js

```diff
@@ -130,6 +135,9 @@ describe('ApiKey Model tests', () => {
       expect(apiKey.getScopes()).to.deep.equal([{
         name: 'imports.write',
         domains: ['https://www.test.com'],
+      },
+      {
+        name: 'imports.delete',
```

Comment:
I think `domains` still pertains directly to the creation, or writing, of import jobs. I don't think it's in a bad place. What benefit would there be to moving it? Easier to update the API keys manually?

Reply:
We could tie domains to the IMSOrg directly. Is it possible that a user from the same org - will have import write permissions to different URLs? He might have write permissions to Url A but not to Url B - though they are both allowed for the given ImsOrg. If that is the case, then I think having domains as part of scopes sounds good.

It will be easier to update it manually. But, if we introduce API Key Management APIs, it probably doesn't matter.